mozetl/tab_spinner/tab_spinner.py

```diff
@@ -1,12 +1,13 @@
 import click
-from pyspark.sql import SparkSession
+from pyspark import SparkConf, SparkContext
 from generate_counts import run_spinner_etl
 
 
 @click.command()
 def main():
-    spark = SparkSession.builder.appName("long_tab_spinners").getOrCreate()
-    run_spinner_etl(spark)
```

Comment:
It's preferable to use the SparkSession, but then use the `sparkContext` attribute.

http://spark.apache.org/docs/latest/api/python/pyspark.sql.html#pyspark.sql.SparkSession.sparkContext

 `run_spinner_etl(spark.sparkContext)`